cityofaustin/atd-product
# Optimize issue fetching

Fixes https://github.com/cityofaustin/atd-data-tech/issues/10465


### Steps to test:
[Deploy preview](https://deploy-preview-39--atd-product.netlify.app/)
Please test everything:
- product list
- projects list, filters, search, chart
- project issue details (all tabs)
- product issue details (all tabs)
- about page

Locally, run `npm run build` and verify that there are no lint errors.
